devito/finite_differences/differentiable.py

```diff
@@ -682,11 +682,12 @@ def __init_finalize__(self, *args, **kwargs):
         assert isinstance(weights, (list, tuple, np.ndarray))
 
         # Normalize `weights`
-        weights = tuple(sympy.sympify(i) for i in weights)
+        from devito.symbolics import pow_to_mul  # noqa, sigh
+        weights = tuple(pow_to_mul(sympy.sympify(i)) for i in weights)
```

Comment:
why?

Reply:
otherwise you get `pow(h_x, 2)` upon reconstructions